packages/docs/site/docs/main/how-to-create-a-theme-using-playground.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Build a WordPress Theme
+slug: /how-to-create-a-theme-using-playground
+---
+
+import ThisIsQueryApi from '@site/docs/\_fragments/\_this_is_query_api.md';
+
+# How to Build a WordPress Theme in WordPress Playground
+
+WordPress Playground is a powerful tool that lets you build, preview, and export WordPress themes—all in your browser or locally via `wp-now`.
+
+This guide walks you through the process step-by-step.
+
+---
+
+## 🚀 1. Creating a New Site
+
+### In the Browser
+
+1. Go to [https://playground.wordpress.net](https://playground.wordpress.net)
+2. Click **"Create New Site"**
+3. Choose WordPress and PHP versions if needed
+4. Click **"Launch"**
+
+### Locally with `wp-now`
```

Comment:
I would remove the wp-now reference and instructions, as it's going to be replaced by Playground CLI. The commands not correct anyway. 

It would be `npx @wp-now/wp-now start`